tests/integration/test_data/generic_e2e/fetch_deps_sha256sums.json

```diff
@@ -1,4 +1,4 @@
- {
-   "generic/archive.zip": "sha256:386428a82f37345fa24b74068e0e79f4c1f2ff38d4f5c106ea14de4a2926e584",
-   "generic/v1.0.0.zip": "sha256:4fbcaa2a8d17c1f8042578627c122361ab18b7973311e7e9c598696732902f87"
- }
+{
+  "generic/archive.zip": "sha256:386428a82f37345fa24b74068e0e79f4c1f2ff38d4f5c106ea14de4a2926e584",
+  "generic/v1.0.0.zip": "sha256:4fbcaa2a8d17c1f8042578627c122361ab18b7973311e7e9c598696732902f87"
+}
```

Comment:
It shouldn't be in this PR, but someone accidentally messed up the formatting.

Reply:
So, can you propose it in a separate PR then?

Reply:
Is it really necessary? It is just 4 spaces. I will add that as the first commit in this PR.

> It shouldn't be in this PR, but someone accidentally messed up the formatting.

I mean by that it was not intended at first :smile:

Reply:
1st in the commit list is okay with me.